Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -915,6 +915,15 @@ void RenderObject::propagateRepaintToParentWithOutlineAutoIfNeeded(const RenderL
     bool repaintRectNeedsConverting = false;
     // Issue repaint on the renderer with outline: auto.
     for (const auto* renderer = this; renderer; renderer = renderer->parent()) {
+        if (renderer->previousSibling() && is<RenderMultiColumnSet>(renderer->previousSibling())) {
+            auto prevMultiColumnSet = downcast<RenderMultiColumnSet>(renderer->previousSibling());
+            auto enclosingMultiColumnFlow = prevMultiColumnSet->multiColumnFlow();
+            auto& prevMultiColumnSetBox = downcast<RenderBox>(*renderer);
```

Comment:
hope we don't move out non-RenderBox type of renderers (I'd test it with a <span> -RenderInline just to make sure this downcast is valid)

Reply:
Verified